posts/test_post_uploading.py

```diff
@@ -16,16 +16,21 @@
 # limitations under the License.
 
 import unittest
+from unittest import mock
 import mongomock
 import app
 
+MOCK_FILE_URL = "the url of an uploaded file"
+MOCK_FILE = mock.MagicMock()
+MOCK_FILE.filename = "the name of a file"
+
 VALID_POST_FULL = {
     "event_id": "abc123",
-    "author_id": "kurt_vonnegut",
-    "text": "This is a very valid post.",
+    "author_id": "jrr_tolkien",
+    "text": "This is a very valid post with text and files.",
     "files": [
-        "Pretend I am a long base-64 encoded file",
-        "Another pretend encoded file"
+        MOCK_FILE,
```

Comment:
Do you need a mock here, or would a simple object work?

Reply:
The 'filename' attribute needs to be accessed. A mock makes this work in 2 lines whereas a class creates a bunch of pain since Pylint flags it for not having enough methods and requires a docstring. Is that preferred?

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/11715c843ba3f11080db72606e658d223bc35adf